examples/astro/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "codehike-astro",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
+  "dependencies": {
+    "@astrojs/mdx": "^0.11.1",
+    "@astrojs/react": "^1.1.3",
+    "@code-hike/mdx": "^0.7.4",
+    "astro": "^1.2.6",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
I was seeing `Warning: Invalid hook call.` warnings in my terminal when running the example. I'm pretty sure its because there are multiple versions of React present in the monorepo.

The warnings went away when I downgraded `react` and `react-dom` to v17, but since this is an example, we probably should use the latest version of React.